src/routes/docs/products/sites/site-hosting/static/+page.markdoc

```diff
@@ -0,0 +1,68 @@
+---
+layout: article
+title: Static
+description: Learn how to host static web apps on Appwrite Sites.
+---
+
+Static apps, also known as static websites, consist of pre-built HTML, CSS, and JavaScript files that are served to users without any backend processing. These apps do not execute server-side code on each request, meaning the content remains the same until manually updated or rebuilt. 
+
+Since the pages are pre-generated, static apps offer incredibly fast load times. However, they lack dynamic interactivity and are best suited for use cases like personal portfolios, documentation sites, and landing pages.
+
+# Configuring your Appwrite Site to use static hosting
+
+When Appwrite builds your site for the first time, it scans your project's configuration files to determine whether the website should be rendered as static pages or using SSR.
+
```

Comment:
Everywhere we ask to change settings on the console a screenshot would be very helpful.

Reply:
I'm making a list of this for Chen, will add them all together once they are ready